tests/install_testbed/create_resources.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -ex
```

Comment:
Delete this line is is not a component that could be executed directly from the CLI.